docs/source/serving/update.rst

```diff
@@ -1,42 +1,56 @@
 .. _serve-update:
 
-Update Your Service
-===========
+Updating a Service
+==================
+
+SkyServe supports *updating* a deployed service, which can be used to change:
+
+* Replica code (e.g., ``run``/``setup``; useful for debugging)
+* Replica resource spec (e.g., number of replicas; accelerator/instance used)
+* Service spec (e.g., autoscaling spec in ``replica_policy``)
 
-SkyServe supports update for your services. Use ``sky serve update`` to update an existing service:
+During an update, the service endpoint will remain the same and the service will
+remain accessible with no downtime.
+
+Use ``sky serve update`` to update an existing service:
 
 .. code-block:: console
 
     $ sky serve update service-name new_service.yaml
 
-SkyServe will launch new replicas described by ``new_service.yaml``. When the number of new replicas reaches the minimum number of replicas (``min_replicas``) required for the service, SkyServe will scale down old replicas to save cost. During the whole process, the service is still accessible to users. SkyServe allows users to update ``replica_policy`` parameters, such as ``target_qps_per_replica``. SkyServe also allows users to update ``resources`` parameters, such as ``cpu`` and ``memory``, so that new replicas can be launched on VMs of different types.   
+SkyServe will launch new replicas described by ``new_service.yaml`` with the following behavior:
+
+* An update is initiated, and traffic will still be redirected to existing (old) replicas.
+* New replicas (with new settings) are brought up in the background.
+* Once ``min_replicas`` new replicas are ready, traffic is *switched*: traffic will be sent to the new
+  replicas, and old replicas will stop receiving traffic and will be scaled down.
+* Replicas are versioned (starting from 1). During an update, traffic is
+  entirely serviced by either old-versioned or new-versioned replicas.
 
-For example, suppose we have a running service hosting Llama 2 model with the following resource configuration: 
+For example, suppose we have a running service hosting a Llama2 model with the following resource configuration:
 
 .. code-block:: yaml
 
     resources:
-        memory: 32+
-        accelerators: T4
+      memory: 32+
+      accelerators: T4
 
-SkyServe supports updating an existing service to a new resource configuration, such as:
+You can update it to use a new resource configuration for each replica, such as:
```

Comment:
This reads a bit like we can potentially have a different resource configuration for each replica.

Reply:
Reworded.